foundry/applications/formApplications/tokenConfig.d.ts

```diff
@@ -1,11 +1,166 @@
 /**
  * A Token Configuration Application
- *
- * @param token - The Token object for which settings are being configured
- * @param options - TokenConfig ui options (see Application)
- *
- * @param configureDefault - Configure the default actor token on submit
  */
-declare class TokenConfig extends FormApplication {
-  _updateObject(event?: Event, formData?: object): Promise<any>; // TODO: Add correct return type
+declare class TokenConfig extends FormApplication<TokenConfig.Data, Token> {
+  /**
+   * @param token   - The Token object for which settings are being configured
+   * @param options - TokenConfig ui options (see Application)
+   */
+  constructor(token: Token, options?: Partial<TokenConfig.Options>);
+
+  options: TileConfig.Options;
+
+  /**
+   * @override
+   * @defaultValue
+   * ```typescript
+   * mergeObject(super.defaultOptions, {
+   *   classes: ["sheet", "token-sheet"],
+   *   template: "templates/scene/token-config.html",
+   *   width: 480,
+   *   height: "auto",
+   *   tabs: [{navSelector: ".tabs", contentSelector: "form", initial: "character"}]
+   * });
+   * ```
+   */
+  static get defaultOptions(): TokenConfig.Options;
+
+  /** @override */
+  get id(): string;
+
+  /**
+   * Convenience access for the Token object
+   */
+  get token(): this['object'];
+
+  /**
+   * Convenience access for the Token's linked Actor, if any
+   */
+  get actor(): this['token']['actor'];
+
+  /** @override */
+  get title(): string;
+
+  /**
+   * @param options - (unused)
+   * @override
+   */
+  getData(options?: Application.RenderOptions): Promise<TokenConfig.Data>;
+
+  /** @override */
+  render(force?: boolean, options?: Application.RenderOptions): Promise<void>;
```

Comment:
```ts
  render(...args: Parameters<FormApplication['render']>): Promise<void>;
```
would be a bit more robust

Reply:
If we do it here, we should do it everywhere, thus probably do it in a separate PR.

Reply:
that's fine with me. I just thought it was a simple small improvment